src/index.js

```diff
@@ -1 +1,27 @@
 'use strict';
+
+import 'dotenv/config';
+import express from 'express';
+import cookieParser from 'cookie-parser';
+import * as routes from './routes/index.js';
+import { errorMiddleware } from './middlewares/index.js';
+
+const PORT = process.env.PORT || 3005;
+
+const app = express();
+
+app.use(express.json());
+app.use(cookieParser());
+
+app.use(routes.authRouter);
```

Comment:
Ensure that `routes.authRouter` is correctly imported and defined in `./routes/index.js`. This is crucial for the application to handle authentication routes properly.